druid/src/scroll_component.rs

```diff
@@ -12,57 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! A container that scrolls its contents.
+//! A component for embedding in another widget to provide consistant and
+//! extendable scrolling behavior
 
-use std::f64::INFINITY;
 use std::time::Duration;
 
 use crate::kurbo::{Affine, Point, Rect, RoundedRect, Size, Vec2};
 use crate::theme;
 use crate::{
-    BoxConstraints, Data, Env, Event, EventCtx, LayoutCtx, LifeCycle, LifeCycleCtx, PaintCtx,
-    RenderContext, TimerToken, UpdateCtx, Widget, WidgetPod,
+    Env, Event, EventCtx, LifeCycle, LifeCycleCtx, PaintCtx, Region, RenderContext, TimerToken,
 };
 
-const SCROLLBAR_MIN_SIZE: f64 = 45.0;
+pub const SCROLLBAR_MIN_SIZE: f64 = 45.0;
```

Comment:
Not sure why this is `pub` now? Might make sense to move this into `Env` instead.

Reply:
I'm definitely in favor of this being customizable, and `Env` sounds like the right place to do it.

Reply:
I agree that this should be moved to `Env` though that should be a follow up PR to address that specifically. It's pub simply due to the process I took when refactoring the logic out of `Scroll`. I can remove the pub if someone really objects but having a const pub feels natural to me anyway and there should be a follow up PR to remove it entirely.

Reply:
A TODO comment has been added to move this to env in a later PR, this PR is already to large as it is